### ISSUE TITLE
default Maximalbreite

### ISSUE THREAD

**Issue by jkoelm** [iqb-berlin/verona-modules-aspect#453]
Auf einer neuen Seite soll das Häkcken bei Maximalbreite gesetzt werden. Die Voreinstellung soll 750 sein.

**Comment by rhenck** [comment]
Ist das wirklich eine geeignete Voreinstellung für alle Benutzer? Das scheint mir zu sehr auf eure speziellen Bedürfnisse zugeschnitten zu sein.

**Comment by jkoelm** [comment]
Ja, das ist eine geeignete Voreinstellung für alle Benutzer. :)

**Comment by rhenck** [comment]
> Ja, das ist eine geeignete Voreinstellung für alle Benutzer. :)

Es scheint mir sinnvoll, die gesamte Bildschirmbreite zu nutzen und sich nicht unnötig zu begrenzen mit einer Seitenbreite. Findest du nicht?
